virt/managing_vms/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -43,6 +44,28 @@ include::modules/virt-running-ssp-pipeline-web.adoc[leveloffset=+2]
 
 include::modules/virt-running-ssp-pipeline-cli.adoc[leveloffset=+2]
 
+[id="virt-deprecated-tasks_{context}"]
+== Removing deprecated or unused resources
+
+You can clean up Tekton-Tasks-Operator resources associated with the operator.
+
+.Procedure
+
+* To remove any remaining Tekton-related resources from the cluster, run the following command:
```

Comment:
```suggestion
* Remove any remaining {pipelines-shortname} resources from the cluster by running the following command:
```
Notes
- I would recommend using the short name attribute for OpenShift Pipelines here. Tekton is the upstream project that the components are based on, but Pipelines is the product name, and seems to be used throughout this assembly.
- Suggested editing for minimalism and to make the step more active